pkg/quay/quay_debug_test.go

```diff
@@ -231,5 +231,101 @@ func TestCreateNotification(t *testing.T) {
 	if !found {
 		t.Fatalf("Notification %s not found", quayNotification.UUID)
 	}
+}
+
+func TestDeleteNotification(t *testing.T) {
```

Comment:
It's not a unit test, please create a test per quay client method as it's done in this file.

Reply:
I don't get your point. I included here _TestDeleteNotification_ and _TestUpdateNotification_ the same as _TestCreateNotification_. They do the same as _TestCreateNotification_ does but for delete and update operations and check several output cases to verify it works well.

Reply:
They are not tests, it's to check if the client implementation works as expected. You can have a smoke test function to test all the notifications cases, but this file is designed to invoke only one method per run (mainly for debug purposes), for example, see how robot accounts handled.
If there is a problem, say, with delete notification, you won't be able to debug it without changing the function (otherwise it will fail on create notification above or it would create another notification that's anyway no desired).